apps/shared/avifutil.h

```diff
@@ -72,6 +72,9 @@ avifAppFileFormat avifReadImage(const char * filename,
                                 avifAppSourceTiming * sourceTiming,
                                 struct y4mFrameIterator ** frameIter);
 
+// Removes a single trailing null character from the image->xmp, if there is exactly one.
+void avifFixImageXMP(avifImage * image);
```

Comment:
Nit: avifFixImageXMP => avifImageFixXMP

to follow libavif's objected-oriented function naming convention.

Alternatively, this function can take &image->xmp as input.